Assets/VRTK/SDK/Daydream/SDK_DaydreamController.cs

```diff
@@ -0,0 +1,266 @@
+﻿// Daydream Controller|SDK_Daydream|003
+namespace VRTK
+{
+#if VRTK_SDK_DAYDREAM
+    using UnityEngine;
+    using System.Collections.Generic;
+
+    public class SDK_DaydreamController : SDK_FallbackController
+    {
+        private GameObject controller;
+
+        private Vector3 prevPosition;
+        private Vector3 prevRotation;
+        private Vector3 velocity = Vector3.zero;
+        private Vector3 angularVelocity = Vector3.zero;
+
+        //TODO: where should we check that controller is connected? eg if (GvrController.State != GvrConnectionState.Connected) 
```

Comment:
Awake() {} is present only to quiet the one inherited from SDK_FallbackController